.github/workflows/main.yml

```
@@ -20,6 +20,8 @@ jobs:
           python -m pip install tox
       - name: Run tests
         run: tox
+      - name: Upload coverage to Codecov
```

If my guess is correct, this will fix the issue.
```suggestion
      - name: Install coveragepy tool
        run: pip install coverage
      - name: Upload coverage to Codecov
```